cmd/gitops/install/cmd.go

```diff
@@ -99,15 +134,42 @@ func installRunCmd(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	_, err = gitopsService.StoreManifests(gitClient, gitProvider, gitopsParams, manifests)
+	cluster := models.Cluster{Name: clusterName}
+	repoWriter := gitrepo.NewRepoWriter(configURL, gitProvider, gitClient, log)
+	automationGen := automation.NewAutomationGenerator(gitProvider, flux, log)
+	gitOpsDirWriter := gitopswriter.NewGitOpsDirectoryWriter(automationGen, repoWriter, osysClient, log)
+
+	clusterAutomation, err := automationGen.GenerateClusterAutomation(ctx, cluster, configURL, namespace)
 	if err != nil {
 		return err
 	}
 
 	if installParams.DryRun {
-		for _, manifest := range manifests {
-			fmt.Println(string(manifest))
+		for _, manifest := range clusterAutomation.Manifests() {
+			log.Println(string(manifest.Content))
 		}
+
+		return nil
+	}
+
+	fluxNamespace, err := kubeClient.FetchNamespaceWithLabel(ctx, LabelPartOf, "flux")
+	if err != nil {
+		return fmt.Errorf("failed getting flux namespace: %w", err)
+	}
+
+	wegoConfigManifest, err := clusterAutomation.GenerateWegoConfigManifest(clusterName, fluxNamespace, namespace)
```

Comment:
@palemtnrider  I'm having seconds thoughts after reading this. I'm starting to think that we don't need to query namespaces to get the one where flux is installed, or at least not yet (there is another ticket for that). I originally didn't call `GenerateWegoConfigManifest` to include it in the dry run logic, because I knew we wouldn't have flux installed, as we would be applying the flux manifests at the end(forgot the initial call to `flux.Intall` with export=false) but out of the dry run logic. What do you think if I used the namespace flag value for now as the "flux namespace" and I include this in that ticket specifically for that second logic?

Reply:
That makes sense to include that work in the following ticket. 👍